### PR TITLE
Added visualization for contract call transactions

### PR DESCRIPTION
**Pull request by stanislav-slavov**

### CODE REVIEW

File: src/popup/router/components/TransactionItem.vue [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
     </div>
     <div class="details">
-      <button v-if="tipUrl" class="url" @click="openUrl(tipUrl, true)">{{ tipUrl }}</button>
-      <span v-else-if="topup || withdraw" class="address">
+      <button
+        v-if="tipUrl || (contractCallData && contractCallData.url)"
+        class="url"
+        @click="openUrl(tipUrl || contractCallData.url, true)"
+      >
+        {{ tipUrl || contractCallData.url }}
```
Comment (davidyuk) [inline src/popup/router/components/TransactionItem.vue:20]: Proposing to add `contractCallData.url` below to how `tipUrl` is calculated, to don't overcomplicate the markup

File: src/popup/router/components/TransactionItem.vue [hunk 2]
```diff
@@ -1,36 +1,55 @@
...
+        />
+        <span :class="status.value">{{ status.text }}</span>
+      </div>
       <span data-cy="time">{{ transaction.microTime | formatDate }}</span>
     </div>
     <div class="details">
-      <button v-if="tipUrl" class="url" @click="openUrl(tipUrl, true)">{{ tipUrl }}</button>
-      <span v-else-if="topup || withdraw" class="address">
+      <button
+        v-if="tipUrl || (contractCallData && contractCallData.url)"
```
Comment (davidyuk) [inline src/popup/router/components/TransactionItem.vue:16]:
```suggestion
        v-if="tipUrl || contractCallData?.url"
```

File: src/popup/router/components/TransactionItem.vue [hunk 3]
```diff
@@ -1,36 +1,55 @@
...
+        @click="openUrl(tipUrl || contractCallData.url, true)"
+      >
+        {{ tipUrl || contractCallData.url }}
+      </button>
+      <span
+        v-else-if="topup || withdraw || (contractCallData && contractCallData.to)"
+        class="address"
+      >
         {{ topup ? transaction.tx.senderId : transaction.tx.recipientId }}
+        {{ contractCallData && contractCallData.to ? contractCallData.to : contractCallData }}
```
Comment (davidyuk) [inline src/popup/router/components/TransactionItem.vue:27]: This getting too complicated, let's extract an `address` computed variable.

File: src/popup/router/components/TransactionItem.vue [hunk 4]
```diff
@@ -1,36 +1,55 @@
...
-      <span v-else-if="topup || withdraw" class="address">
+      <button
+        v-if="tipUrl || (contractCallData && contractCallData.url)"
+        class="url"
+        @click="openUrl(tipUrl || contractCallData.url, true)"
+      >
+        {{ tipUrl || contractCallData.url }}
+      </button>
+      <span
+        v-else-if="topup || withdraw || (contractCallData && contractCallData.to)"
```
Comment (davidyuk) [inline src/popup/router/components/TransactionItem.vue:23]:
```suggestion
        v-else-if="topup || withdraw || contractCallData?.to"
```

**Review by davidyuk — APPROVED** [review]

**Review by mradkov — APPROVED** [review]

**Pull request merged**